qmf/src/plugins/ssoauth/password/passwordplugin.cpp

```diff
@@ -94,20 +94,22 @@ QMap<QString, QList<QByteArray> > SSOPasswordPlugin::getIMAPAuthentication(const
     QMap<QString, QList<QByteArray> > result;
 
     // Add PLAIN auth
-    QByteArray user(username.toLatin1());
-    QByteArray pass(password.toLatin1());
+    QByteArray user(username.toUtf8());
```

Comment:
This isn't legal, is it?  If supported, it would have to encoded into quoted-prinatble or something, surely?